asab/web/authz/service.py

```diff
@@ -1,8 +1,13 @@
-import hashlib
+import json
+import logging
 import aiohttp
 
+import jwcrypto.jwk
+import jwcrypto.jwt
+import jwcrypto.jws
```

Comment:
Be careful about this dependency.

Reply:
should i add it as a optional dependency to setup.py? or should i avoid jwcrypto altogether and try to parse and verify the token manually?